kiliautoml/utils/helpers.py

```diff
@@ -122,11 +124,17 @@ def get_assets(
     assets = get_asset_memoized(
         kili=kili,
         project_id=project_id,
-        first=max_assets,
+        total=max_assets,
         skip=0,
         status_in=status_in,
     )
 
+    # In order to obtain a mix of all assets, we need to shuffle the list
+    random.shuffle(assets)
```

Comment:
if we do not fix the seed, Trying 2 times to train the model will lead to different results since the shuffle will be different, isn't it?

Reply:
The seed is fixed